src/js/tech/html5.js

```diff
@@ -14,6 +14,7 @@ import document from 'global/document';
 import window from 'global/window';
 import assign from 'object.assign';
 import mergeOptions from '../utils/merge-options.js';
+import TitleCase from '../utils/to-title-case.js';
```

Comment:
import toTitleCase from '../../utils/to-title-case.js'; (same as all imports in videojs)
https://github.com/videojs/video.js/search?utf8=%E2%9C%93&q=toTitleCase